<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite name="TestCases.Regression.SalesForce_3rdPartyAddressValidation" tests="1" time="113.166" ignored="7" failures="0" timestamp="20 May 2019 05:20:26 GMT" skipped="0" hostname="Av-Pn-Neha" errors="1">
  <testcase name="TestCase162_USAddressValidated" time="113.166" classname="TestCases.Regression.SalesForce_3rdPartyAddressValidation">
    <error type="org.openqa.selenium.TimeoutException" message="Expected condition failed: waiting for element to be clickable: By.xpath: //div[@class=&#039;pbHeader&#039;]//input[@name=&#039;avacloud_sand__order_validate_shipping_address&#039;] (tried for 60 second(s) with 500 MILLISECONDS interval)">
      <![CDATA[org.openqa.selenium.TimeoutException: Expected condition failed: waiting for element to be clickable: By.xpath: //div[@class='pbHeader']//input[@name='avacloud_sand__order_validate_shipping_address'] (tried for 60 second(s) with 500 MILLISECONDS interval)
	at org.openqa.selenium.support.ui.WebDriverWait.timeoutException(WebDriverWait.java:82)
	at org.openqa.selenium.support.ui.FluentWait.until(FluentWait.java:231)
	at GenericFunctionLibrary.genericFunctions.expectelementToBeClickable(genericFunctions.java:800)
	at FunctionalLibrary.functionLibrary.fTriggerShipToAddress(functionLibrary.java:963)
	at TestCases.Regression.SalesForce_3rdPartyAddressValidation.addressValidation(SalesForce_3rdPartyAddressValidation.java:107)
	at TestCases.Regression.SalesForce_3rdPartyAddressValidation.TestCase162_USAddressValidated(SalesForce_3rdPartyAddressValidation.java:52)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:104)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:645)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:851)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1177)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:129)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:112)
	at org.testng.TestRunner.privateRun(TestRunner.java:756)
	at org.testng.TestRunner.run(TestRunner.java:610)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:387)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:382)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:340)
	at org.testng.SuiteRunner.run(SuiteRunner.java:289)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1293)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1218)
	at org.testng.TestNG.runSuites(TestNG.java:1133)
	at org.testng.TestNG.run(TestNG.java:1104)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:114)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
Caused by: org.openqa.selenium.NoSuchElementException: Cannot locate an element using By.xpath: //div[@class='pbHeader']//input[@name='avacloud_sand__order_validate_shipping_address']
For documentation on this error, please visit: http://seleniumhq.org/exceptions/no_such_element.html
Build info: version: '3.7.0', revision: '2321c73', time: '2017-11-02T22:22:35.584Z'
System info: host: 'AV-PN-NEHA', ip: '10.240.10.58', os.name: 'Windows 10', os.arch: 'amd64', os.version: '10.0', java.version: '1.8.0_144'
Driver info: driver.version: unknown
	at org.openqa.selenium.support.ui.ExpectedConditions.lambda$findElement$0(ExpectedConditions.java:882)
	at java.util.Optional.orElseThrow(Unknown Source)
	at org.openqa.selenium.support.ui.ExpectedConditions.findElement(ExpectedConditions.java:881)
	at org.openqa.selenium.support.ui.ExpectedConditions.access$000(ExpectedConditions.java:43)
	at org.openqa.selenium.support.ui.ExpectedConditions$7.apply(ExpectedConditions.java:205)
	at org.openqa.selenium.support.ui.ExpectedConditions$7.apply(ExpectedConditions.java:201)
	at org.openqa.selenium.support.ui.ExpectedConditions$22.apply(ExpectedConditions.java:643)
	at org.openqa.selenium.support.ui.ExpectedConditions$22.apply(ExpectedConditions.java:640)
	at org.openqa.selenium.support.ui.FluentWait.until(FluentWait.java:208)
	... 29 more
]]>
    </error>
  </testcase> <!-- TestCase162_USAddressValidated -->
  <testcase name="TestCase168_NonDeliverableAddress" classname="TestCases.Regression.SalesForce_3rdPartyAddressValidation">
    <skipped/>
  </testcase> <!-- TestCase168_NonDeliverableAddress -->
  <testcase name="TestCase166_AddressOutOfRange" classname="TestCases.Regression.SalesForce_3rdPartyAddressValidation">
    <skipped/>
  </testcase> <!-- TestCase166_AddressOutOfRange -->
  <testcase name="TestCase169_CityError" classname="TestCases.Regression.SalesForce_3rdPartyAddressValidation">
    <skipped/>
  </testcase> <!-- TestCase169_CityError -->
  <testcase name="TestCase165_NonUS_NonCANAddress" classname="TestCases.Regression.SalesForce_3rdPartyAddressValidation">
    <skipped/>
  </testcase> <!-- TestCase165_NonUS_NonCANAddress -->
  <testcase name="TestCase167_UnknownStreet" classname="TestCases.Regression.SalesForce_3rdPartyAddressValidation">
    <skipped/>
  </testcase> <!-- TestCase167_UnknownStreet -->
  <testcase name="TestCase163_CANAddressValidated" classname="TestCases.Regression.SalesForce_3rdPartyAddressValidation">
    <skipped/>
  </testcase> <!-- TestCase163_CANAddressValidated -->
  <testcase name="TestCase164_MilitaryAddressValidated" classname="TestCases.Regression.SalesForce_3rdPartyAddressValidation">
    <skipped/>
  </testcase> <!-- TestCase164_MilitaryAddressValidated -->
</testsuite> <!-- TestCases.Regression.SalesForce_3rdPartyAddressValidation -->
